platform/bitcoin/base.go

```diff
@@ -2,18 +2,19 @@ package bitcoin
 
 import (
 	"github.com/trustwallet/blockatlas/internal"
+	"github.com/trustwallet/blockatlas/platform/ethereum/blockbook"
```

Comment:
move `blockbook` out of ethereum folder?

Reply:
how about move into `bitcoin` folder?

Reply:
`pkg/blockbook`?